src/main/java/games/strategy/triplea/delegate/MustFightBattle.java

```diff
@@ -379,15 +379,15 @@ public void fight(final IDelegateBridge bridge) {
       // play a sound
       if (Match.anyMatch(m_attackingUnits, Matches.UnitIsSea)
           || Match.anyMatch(m_defendingUnits, Matches.UnitIsSea)) {
-        if (Match.allMatchNotEmpty(m_attackingUnits, Matches.UnitIsSub)
+        if (!m_attackingUnits.isEmpty() && Match.allMatch(m_attackingUnits, Matches.UnitIsSub)
```

Comment:
Parentheses are not required here, but it may make it easier to read due to the `||`.